include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -198,6 +198,14 @@ SECTIONS
 #include <linker/priv_stacks-text.ld>
 #include <linker/kobject-text.ld>
 
+	/* TI CC13x2 ROM RO Data Region */
```

Comment:
Based on what I am seeing in the nortos examples in the TI SimpleLink SDK, I wouldn't have expected these data regions to be necessary. I am trying to confirm these with a TI engineer - will keep you posted.

Reply:
I got the confirmation today that these sections are indeed TIRTOS-specific, and are only needed when building against TI's kernel in ROM. So they are not applicable when running in Zephyr.